WPMedia/fusion-news-theme-blocks
# PEN-1507 Simple List title should not be h2-tag

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
_Information about what you changed for this PR_
replace h2 tag by using div on Simple list title

## Jira Ticket
- [PEN-1507](https://arcpublishing.atlassian.net/browse/PEN-1507)

## Acceptance Criteria
_copy from ticket_
When looking at a Simple List here:  http://liberation-liberation-sandbox.cdn.arcpublishing.com/ (right rail: Latest News)

You can see that the "Latest News" (or whatever else is used) has an h2-tag, but it should not be an h2 tag as this title is not relevant for SEO reasons

## Test Steps
- Add test steps a reviewer must complete to test this PR
1. In Fusion-News-Theme repo, checkout master & git pull
2. After checking out this feature branch in fusion-news-theme-blocks, run rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..
3. Run npx fusion start -f -l @wpmedia/simple-list-block in Fusion-News-Theme
4. Create a page then add Simple List for testing purpose
Inspect the html element on simple list title and verify html tag

## Effect Of Changes
### Before
_Example: When I clicked the search button, the button turned red._
![image](https://user-images.githubusercontent.com/61674931/101132756-1ac42f80-363a-11eb-93ff-d02d207d3cac.png)

[include screenshot or gif or link to video, storybook would be awesome]

### After
_Example: When I clicked the search button, the button turned green._
![image](https://user-images.githubusercontent.com/61674931/101133277-fddc2c00-363a-11eb-9d1d-64e39cafc7ac.png)

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
